buildserver/vfs.go

```diff
@@ -26,8 +29,16 @@ var RemoteFS = func(ctx context.Context, initializeParams lspext.InitializeParam
 		if !ok {
 			return ""
 		}
-		url, _ := initializationOptions["zipURL"].(string)
-		return url
+		zipURL, _ := initializationOptions["zipURL"].(string)
+		if zipURL != "" {
+			return zipURL
+		}
+		zipURLTemplate, _ := initializationOptions["zipURLTemplate"].(string)
+		root, err := url.Parse(string(initializeParams.OriginalRootURI))
+		if err != nil {
+			return ""
+		}
+		return fmt.Sprintf(zipURLTemplate, path.Join(root.Host, root.Path), root.RawQuery)
```

Comment:
you may want to test this with an empty zipURLTemplate.

Reply:
Ah, `fmt.Sprintf("", ...)` results in something like `%!(EXTRA string=foo)`. I added a check for this.